packages/ppx/test/bucklescript/src/media_query_test.re

```diff
@@ -48,7 +48,7 @@ let testData = [
            and (orientation: portrait) { color: brown; }"],
     CssJs.style(. [|
       CssJs.media(.
-        "all and (max-width: 218px) and (max-height: 281px) and (orientation: portrait)",
+        "all and (max-width: 218px) and (max-height: 281px) and (orientation: portrait) ",
```

Comment:
Maybe we should String.trim the selector?

Reply:
I tried to apply `String.trim`, but different from the string concatenation, bucklescript/rescript doesnt inline the result, so the output would be more verbose (`CssJs.media($$String.trim("all and (max-width: 218px) and (max-height: 281px) and (orientation: portrait) ")`) and instead of adding a whitespace we would add `String.trim` to all media tests.

Do you prefer to change it or have other suggestion on how to do it?

Reply:
It's not really critical to be honest, but looks dirty. Let's not block this PR because of this, we can revisit this.